extension/intents/tabs/tabs.toml

```diff
@@ -262,3 +262,27 @@ match = """
 
 [[tabs.moveSelectedToNewWindow.example]]
 phrase = "Move selected tabs to new window"
+
+[tabs.collectAllTabs]
+description = "Collect all tabs by url"
+match = """
+  collect (all | the|) tabs in all (the|) windows [currentWindow=false]
+  collect (all | the|) tabs [currentWindow=true]
+"""
+
+[[tabs.collectAllTabs.example]]
+phrase = "Collect all tabs"
+
+[tabs.collectMentionedTabs]
+description = "Collect tabs similar to active tab or by query"
+match = """
+  collect these tabs [currentWindow=true]
+  collect these tabs in all (the|) windows [currentWindow=false]
```

Comment:
Also I think this could match `collect tabs (like | from |) (this |) (site | page | url | domain | host)` and then another version from all windows.

And (in | from) here as well